src/ThemeDecorator.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import {ThemeProvider} from './ThemeProvider'
+import BaseStyles from './BaseStyles'
+
+export const ThemeDecorator = (Component: React.FC) => {
```

Comment:
Could you add a comment here explaining what this component is and how it should be used?

Reply:
This isn't the right location for this (closed PR, will open new one soon), but the idea was to export what we have in [.storybook/preview.js](https://github.com/primer/react/blob/main/.storybook/preview.js#L116) from primer/react so that it can be used to decorate stories with [primer/react-scripts](https://github.com/primer/react-scripts)